types/node/util.d.ts

```diff
@@ -1105,6 +1105,15 @@ declare module 'util' {
          */
         encodeInto(src: string, dest: Uint8Array): EncodeIntoResult;
     }
+
+    global {
+        /**
+         * `TextEncoder` class is a global reference for `require('util').TextEncoder`
+         * https://nodejs.org/api/globals.html#textencoder
+         * @since v11.0.0
+         */
+         const TextEncoder: typeof import('util').TextEncoder;
+    }
```

Comment:
```suggestion
    global {
        /**
         * `TextEncoder` class is a global reference for `require('util').TextEncoder`
         * https://nodejs.org/api/globals.html#textencoder
         * @since v11.0.0
         */
        var TextEncoder: {
            prototype: TextEncoder;
            new (): TextEncoder;
        };
    }
```
that will remove current errors. Is that symbol export you'd expect here?

Reply:
Sorry I was still running tests last night so I didn't get to commit my fixes.  This should be OBE with the commit I pushed -- TextEncoder is just one of the DOM-like globals that have been added in the last couple of Node versions.  The updated branch should expose all the non-experimental global constructors conditionally (but none of the matching interfaces).